src/app/portlets/content-types/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="content-type-fields-drop-zone__container"
```

Comment:
We need to come up with something on how are we going to handle the format of the attributes... because I'm seeing different things all over the code.